### DISCUSSION TITLE
Update to snippets:how-to-install-multiple-instances-of-ubuntu-in-wsl2

### DISCUSSION THREAD

**Discussion by the-moog** [CloudBytes-Academy/CloudBytes.dev#140] [General]
[This post](https://cloudbytes.dev/snippets:how-to-install-multiple-instances-of-ubuntu-in-wsl2) by by @rehanhaider  [Rehan Haider](https://cloudbytes.dev/authors/rehan-haider), Thu 11 November 2021. was very helpful but I should point out that the URLs given for download of WSL2 root file systems for Ubuntu have changed.

Were located at http://cloud-images.ubuntu.com/releases/....
are
Now located at [https://cloud-images.ubuntu.com/wsl/...](https://cloud-images.ubuntu.com/wsl/)
With folders for
- [jammy(22.04)](https://cloud-images.ubuntu.com/wsl/jammy/)
- [kinetic(22.10)](https://cloud-images.ubuntu.com/wsl/kinetic/)
- [lunar(23.04)](https://cloud-images.ubuntu.com/wsl/lunar/)
- [mantic(23.10)](https://cloud-images.ubuntu.com/wsl/mantic/)

Each folder having several datepoint releases `[YYYYMMDD]` with `current` being a symlink to the latest.
e.g. [https://cloud-images.ubuntu.com/wsl/lunar/current](https://cloud-images.ubuntu.com/wsl/lunar/current)
The TGZ filename has changed as well.

and so the curl command would look like
```
curl (("https://cloud-images.ubuntu.com", "wsl/lunar/current", "ubuntu-lunar-wsl-amd64-wsl.rootfs.tar.gz") -join "/") --output ubuntu-23.04-wsl-rootfs-tar.gz
```

**Comment by rehanhaider** [comment]
> [This post](https://cloudbytes.dev/snippets:how-to-install-multiple-instances-of-ubuntu-in-wsl2) by by @rehanhaider [Rehan Haider](https://cloudbytes.dev/authors/rehan-haider), Thu 11 November 2021. was very helpful but I should point out that the URLs given for download of WSL2 root file systems for Ubuntu have changed.
> 
> Were located at http://cloud-images.ubuntu.com/releases/.... are Now located at [https://cloud-images.ubuntu.com/wsl/...](https://cloud-images.ubuntu.com/wsl/) With folders for
> 
> * [jammy(22.04)](https://cloud-images.ubuntu.com/wsl/jammy/)
> * [kinetic(22.10)](https://cloud-images.ubuntu.com/wsl/kinetic/)
> * [lunar(23.04)](https://cloud-images.ubuntu.com/wsl/lunar/)
> * [mantic(23.10)](https://cloud-images.ubuntu.com/wsl/mantic/)
> 
> Each folder having several datepoint releases `[YYYYMMDD]` with `current` being a symlink to the latest. e.g. https://cloud-images.ubuntu.com/wsl/lunar/current The TGZ filename has changed as well.
> 
> and so the curl command would look like
> 
> ```
> curl (("https://cloud-images.ubuntu.com", "wsl/lunar/current", "ubuntu-lunar-wsl-amd64-wsl.rootfs.tar.gz") -join "/") --output ubuntu-23.04-wsl-rootfs-tar.gz
> ```

The link still seems to be working though.

**Reply by the-moog** [reply]
@rehanhaider If you look at the folder tree from the old link, only one WSL rootfs revision is still there (21.10) all other revisions are moved to the new path.

**Reply by rehanhaider** [reply]
Okay, would you be so kind to make the changes and raise a PR?

**Reply by the-moog** [reply]
You know what? I did not realise that the original post (which is now gone anyway) was in the form of a github repo. I assumed it was some sort of online magazine -which is is!! Anyway I guess this was sorted ages ago. Sorry for not replying sooner. I just noticed your message during a cleanup. It had fallen off the end.

**Reply by rehanhaider** [reply]
The [post](https://cloudbytes.dev/snippets/how-to-install-multiple-instances-of-ubuntu-in-wsl2) is still online, the URL you had was misformatted
